app/routes/reductions.js

```diff
@@ -133,17 +134,65 @@ module.exports = function (router) {
     })
   })
 
+  router.post('/:organisationLevel/:id/update-reduction-status', function (req, res, next) {
+    var organisationLevel = req.params.organisationLevel
+
+    if (organisationLevel !== organisationUnitConstants.OFFENDER_MANAGER.name) {
+      throw new Error('Only available for offender manager')
+    }
+
+    var reductionStatus = req.body.status
+    var id = req.params.id
+    var reductionId = req.body.reductionId
+
+    if (!requestStatusVerified(reductionStatus)) {
+      return res.redirect(302, '/' + organisationLevel + '/' + id + '/reductions?fail=true')
+    }
+
+    return reductionsService.updateReductionStatus(id, reductionId, reductionStatus)
+    .then(function () {
+      return res.redirect(302, '/' + organisationLevel + '/' + id + '/reductions?success=true')
+    })
+  })
+
   var generateNewReductionFromRequest = function (requestBody) {
     var reductionStartDate = new Date(requestBody.red_start_year, requestBody.red_start_month - 1, requestBody.red_start_day)
     var reductionEndDate = new Date(requestBody.red_end_year, requestBody.red_end_month - 1, requestBody.red_end_day)
+    var reductionStatus = setReductionStatus(reductionStartDate, reductionEndDate)
 
     return new Reduction(
-      requestBody.reasonForReductionId, requestBody.hours, reductionStartDate, reductionEndDate, requestBody.notes
+      requestBody.reasonForReductionId, requestBody.hours, reductionStartDate, reductionEndDate, requestBody.notes, reductionStatus
     )
   }
 
+  var setReductionStatus = function (reductionStartDate, reductionEndDate) {
+    var currentDate = new Date()
+    var status = reductionStatusType.ACTIVE
+    if ((reductionStartDate < currentDate) && (reductionEndDate > currentDate)) {
+      status = reductionStatusType.ACTIVE
+    } else if ((reductionStartDate > currentDate) && (reductionEndDate > currentDate)) {
+      status = reductionStatusType.SCHEDULED
+    } else if ((reductionStartDate < currentDate) && (reductionEndDate < currentDate)) {
+      status = reductionStatusType.ARCHIVED
+    }
+
+    return status
+  }
+
+  var requestStatusVerified = function (reductionStatus) {
+    if (reductionStatus !== reductionStatusType.ACTIVE &&
```

Comment:
Could you write this as an includes maybe?  eg
eg statuses = [reductionStatusType.ACTIVE, ...]
if(!statues.includes(reductionStatus){ do stuff }

Might be easier to read, but it might not, only a suggestion